common/src/sql/dbinit.sql

```diff
@@ -108,13 +108,16 @@ CREATE INDEX ON omicron.public.sled (
  */
 
 CREATE TYPE omicron.public.service_kind AS ENUM (
+  'crucible_pantry',
+  'dendrite',
+  'external_dns_config',
+  'external_dns',
+  'internal_dns_config',
   'internal_dns',
```

Comment:
Ah, so the expansion here is because these have the same IP but different ports?

Reply:
Basically yes, though it's not just an artifact of only supporting one port in the database.  Logically they're two different things -- different consumers, different protocol, different APIs, etc.